test/functional/p2p_mutated_blocks.py

```diff
@@ -79,7 +83,10 @@ def self_transfer_requested():
 
         # Attempt to clear the honest relayer's download request by sending the
         # mutated block (as the attacker).
-        with self.nodes[0].assert_debug_log(expected_msgs=["Block mutated: bad-txnmrklroot, hashMerkleRoot mismatch"]):
+        with self.nodes[0].assert_debug_log(
+          expected_msgs=["Block mutated: bad-txnmrklroot, hashMerkleRoot mismatch"],
+          # header is known - mutated block isn't logged
+          unexpected_msgs=[LOGMESSAGE_REJECTED_MUTATED_BLOCK]):
```

Comment:
`unexpected_msgs` is unused

Reply:
Oh oops, nvm

Reply:
In that case, it's good that you're also testing below that `LOGMESSAGE_REJECTED_MUTATED_BLOCK` _is_ logged. Otherwise if the string is changed in the code, we would not catch later regressions (where it shows up in the log).